cms/templates/widgets/footer.html

```diff
@@ -33,9 +32,11 @@
                 <a data-rel="edx.org" href="${reverse('accessibility')}">${_("Accessibility Accommodation Request")}</a>
               </li>
             %endif
+            <li class="nav-item">
+              <a data-rel="edx.org" href=${settings.LMS_ROOT_URL}>${_("LMS")}</a>
```

Comment:
If it's not too late, i'd like to request one more feature:  Let's give this tag an id:
```
<a href="${settings.LMS_ROOT_URL}" id="lms-link">
```
(BTW: I just noticed you are missing quotes around the url also....!)

Reply:
Would you like to keep the data-rel attribute on it? It doesn't seem like it's in use anywhere in edx-platform.

I have added the id attribute and fixed the missing quotation marks. Thank you.

One thing I'm not certain about is whether LMS_ROOT_URL is universally the right setting to use to refer to the LMS URL? I noticed that it does not work correctly on Devstack, since it links to "http://edx.devstack.lms:18000/" and not "localhost:18000". That's what it's in the [env file|https://github.com/edx/edx-platform/blob/master/lms/envs/devstack_docker.py#L15]. I'm not sure if edx.devstack.lms is supposed to resolve to localhost or something and it's not, or if I'm missing something. Perhaps it's just a Devstack specific thing, but I wanted to flag it, just in case.

Reply:
In Studio, isn't there a "View Live" link when authoring content that takes you to the page in the LMS?  Not Preview, but a link to the real site?  If so, does it use LMS_ROOT_URL?

Reply:
Yes, there is. I took a look at the "View Live" button from the Course Outline page, and it has the same issue. It links, from the Demo Course, to "http://edx.devstack.lms:18000/courses/course-v1:edX+DemoX+Demo_Course/jump_to/block-v1:edX+DemoX+Demo_Course+type@course+block@course".

The anchor tag is in this [course_outline.html|https://github.com/edx/edx-platform/blob/9abec119fc1edc045e8bda8bc97c6e57ad50739f/cms/templates/course_outline.html#L142] file. The `lms_link` is returned by the get_lms_link_for_item [utility function|https://github.com/edx/edx-platform/blob/9abec119fc1edc045e8bda8bc97c6e57ad50739f/cms/djangoapps/contentstore/views/course.py#L614], which is defined [here|https://github.com/edx/edx-platform/blob/9abec119fc1edc045e8bda8bc97c6e57ad50739f/cms/djangoapps/contentstore/utils.py#L98]. It looks like it will use the SiteConfiguration's LMS_BASE value, if it exists, and the settings.LMS_BASE value otherwise. I'm not familiar with SiteConfigurations. Is that something to be concerned about?

Reply:
If this does the same thing as View Live, then we are good to go, and devstack might have an issue.